packages/shared-preferences/ios/shared_preferences.podspec

```diff
@@ -0,0 +1,21 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
+#
+Pod::Spec.new do |s|
+  s.name             = 'shared_preferences'
+  s.version          = '0.0.1'
+  s.summary          = 'A new flutter plugin project.'
+  s.description      = <<-DESC
```

Comment:
I think we have agreed not to maintain this. We only have it to make CocoaPods happy.